src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/widget/WeiboWidgetPageObject.java

```diff
@@ -11,12 +13,17 @@
 
   @FindBy(css = "iframe[data-wikia-widget='weibo']")
   private WebElement weiboIframe;
+  @FindBy(css = "iframe[data-wikia-widget='weibo']")
+  private List<WebElement> weiboIframeList;
   @FindBy(css = "div.tsina_open")
   private WebElement weiboBody;
 
   private static final String TAG_NAME = "weibo";
   private static final String ARTICLE_NAME = "WeiboWidget";
-  private static final String TAG = "<weibo uids=\"1642909335,1782515283\" />";
+  private static final String TAGS[] = {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move the array designator from the variable to the type. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1197)